### DISCUSSION TITLE
Computed properties with camel case names not working

### DISCUSSION THREAD

**Discussion by rudolfbruder** [livewire/livewire#5661] [1. Help] [answered]
Hi,

I am facing a problem where when I am using computed properties written in camel case (let's say variable should be called WithdrawalReasons) the computed property does not work.

This does not work:

```
    public function getWithdrawalReasonsProperty()
    {
        return Cache::remember('withdrawal_reasons', config('cache.durations.day'), function () {
            return WithdrawalReason::with('currentLangTranslation')->orderBy('rank')->get();
        });
    }
```

This does work:

```
    public function getWithdrawalreasonsProperty()
    {
        return Cache::remember('withdrawal_reasons', config('cache.durations.day'), function () {
            return WithdrawalReason::with('currentLangTranslation')->orderBy('rank')->get();
        });
    }
```

Therefor I have to use $this->withdrawalreasons in the blade views.

Is there some kind of solution please?

Thanks.

**Comment by joshhanley** [comment, marked as answer]
@rudolfbruder your first example will work, but it should be true camelCase when you call it, so try `$this->withdrawalReasons`.

What you're trying to do is PascalCase, but Livewire doesn't support that only camelCase.

Hope this helps!

**Reply by rudolfbruder** [reply]
All right I guess my bad, I just got confused with getwithdrawalReasonProperty - you know getw.. being small but I get it that it's a 'getter' let me try it out. cheers
